--- node_modules/hangupsjs/lib/client.js.orig	2015-11-12 02:37:01.555846529 +0900
+++ node_modules/hangupsjs/lib/client.js	2015-11-12 02:37:16.535875103 +0900
@@ -37,7 +37,7 @@
 
   ref1 = require('./schema'), OffTheRecordStatus = ref1.OffTheRecordStatus, TypingStatus = ref1.TypingStatus, ClientNotificationLevel = ref1.ClientNotificationLevel, CLIENT_SYNC_ALL_NEW_EVENTS_RESPONSE = ref1.CLIENT_SYNC_ALL_NEW_EVENTS_RESPONSE, CLIENT_GET_CONVERSATION_RESPONSE = ref1.CLIENT_GET_CONVERSATION_RESPONSE, CLIENT_GET_ENTITY_BY_ID_RESPONSE = ref1.CLIENT_GET_ENTITY_BY_ID_RESPONSE;
 
-  IMAGE_UPLOAD_URL = 'http://docs.google.com/upload/photos/resumable';
+  IMAGE_UPLOAD_URL = 'https://docs.google.com/upload/photos/resumable';
 
   DEFAULTS = {
     rtokenpath: syspath.normalize(syspath.join(__dirname, '../refreshtoken.txt')),
